src/Jackett.Common/Services/ConfigurationService.cs

```diff
@@ -81,33 +81,44 @@ public void CreateOrMigrateSettings()
                     }
                     else
                     {
-                        PerformMigration();
+                        PerformMigration(oldDir);
                     }
-
                 }
                 catch (Exception e)
                 {
                     logger.Error($"ERROR could not migrate settings directory\n{e}");
                 }
             }
+
+            // Perform a migration in case of https://github.com/Jackett/Jackett/pull/11173#issuecomment-787520128
+            if (Environment.OSVersion.Platform == PlatformID.Unix)
+            {
+                var oldDirectory = "Jackett";
+                if (Directory.Exists(oldDirectory))
```

Comment:
Checking if the directory exists already happens in PerformMigration.

Reply:
Address in 1ade5585761a6e206580533ca99bed3d73d93625